### ISSUE TITLE
Tab key is trapped inside CKFinder

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#247]
1. Open the widget sample in Chrome.
2. Start tabbing through links etc.
3. When CKFinder is reached, one cannot move out of CKFinder using `tab` / `shift+tab`.

Expected result: it should be possible to leave CKFinder with tab key. E.g. when status bar is focused and tab key is pressed ("last element" in tab order), or when toolbar is focused ("first element" in tab order) and shift+tab is pressed.

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
